deepchecks/nlp/checks/data_integrity/text_property_outliers.py

```diff
@@ -129,7 +136,8 @@ def run_logic(self, context: Context, dataset_kind: DatasetKind) -> CheckResult:
                 # we have in the data
                 'lower_limit': max(lower_limit, min(values_arr)),
                 'upper_limit': min(upper_limit, max(values_arr)) if is_numeric else None,
-                'outlier_ratio': len(text_outliers) / len(values_arr)
+                'outlier_ratio': len(text_outliers) / len(values_arr),
+                'property_error': property_error
```

Comment:
would change this, 
if a property outlier calculation fails - it shouldn't have the same structure in the result dict
specifically after you add the try - catch
under the catch would set result[name] = 'exception error msg'

Reply:
Yes, added try-except block